composables/store/project.ts

```diff
@@ -81,32 +81,58 @@ export const useProjectStore = defineStore('project', () => {
   };
 
   const setSelected = (id: string, isSelected: boolean) => {
-    if (isSelected) {
-      const rowId = getObjectRow.value(id);
-      const row = getRow.value(rowId);
-      const obj = getObject.value(id);
-      if (row.allowedChoices > 0 && !obj.isSelectableMultiple) {
-        const selectedRowObjects = R.intersection(
-          R.keys(selected.value),
-          R.map(R.prop('id'), row.objects),
-        );
-
-        if (selectedRowObjects.length >= row.allowedChoices) {
-          const toDeselect = selectedRowObjects[0];
-          selected.value = R.pipe(
-            R.dissoc(toDeselect),
-            R.assoc(id, 1),
-          )(selected.value);
-        } else {
-          selected.value = R.assoc(id, 1, selected.value);
-        }
+    let selectedN = R.clone(selected.value);
+    // Add or remove object from selection based on isSelected
+    // function for reuse
+    const addOrRemove = (objId: string, AddToSelected: boolean, num = 1) => {
```

Comment:
This probably can be moved, but I wasn't sure where, or if it was worth being a standalone function.